Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu
+WORKDIR /root
+
+RUN sed -i '/path-exclude=\/usr\/share\/man\/*/c\#path-exclude=\/usr\/share\/man\/*' /etc/dpkg/dpkg.cfg.d/excludes
+
+RUN apt-get update -qq && \
+    apt-get install -y man \
+                       manpages-posix \
+                       man-db \
+                       vim \
+                       screen \
+                       curl \
+                       jq \
+                       docker.io \
```

Comment:
TIL, is this the new preferred way to install docker on ubuntu? I always thought getting it from docker's repository was the recommended approach.